app/User.php

```diff
@@ -28,6 +28,14 @@ class User extends Authenticatable
         'password', 'remember_token',
     ];
 
-    protected $test = 'testtest';
-    protected $test2 = 'test2test2';
+    private $test = 'testtest';
+    private $test2 = 'test2test2';
+
+    private function testWIP() {
+        $test = 'failed';
+        return $test;
+        $test = 'test again';
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "test again" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1192)